fr/development/testing.rst

```diff
@@ -1063,6 +1063,64 @@ toutes les assertions de `TestSuite
 de
 `PHPUnit <https://phpunit.de/manual/current/en/appendixes.assertions.html>`__.
 
+Comparer les Résultats du Test avec un Fichier
+----------------------------------------------
+
+Pour certains types de test, il peut être plus simple de comparer les résultats
+d'un test avec le contenu d'un fichier - par exemple, quand vous testez la
+sortie rendue d'une view.
+``StringCompareTrait`` ajoute une méthode d'assertion simple pour cela.
+
+L'utilisation implique l'utiisation d'un trait, définissant le chemin de base
```

Comment:
"L'utilisation implique l'utilisation"

I'd go with : "Pour l'utiliser, vous devez charger (ou inclure, au choix) un Trait, définir le chemin de base de comparaison et appeler ``assertSameAsFile``"

What do you think ?